mungegithub/mungers/approval-handler.go

```diff
@@ -231,7 +275,7 @@ func createApproverSet(comments []*githubapi.IssueComment) sets.String {
 	approverMatcher := c.CommandName(approveCommand)
 	for _, comment := range c.FilterComments(comments, approverMatcher) {
 		cmd := c.ParseCommand(comment)
-		if cmd.Arguments == "cancel" {
+		if cmd.Arguments == cancel {
```

Comment:
Never mind, I checked that it is a NOP if that is the case. This looks good to go.